lib/modules/manager/conan/__fixtures__/conanfile.txt

```diff
@@ -29,4 +30,4 @@ openssl:shared=True
 
 [imports]
 bin, *.dll -> ./bin # Copies all dll files from packages bin folder to my local "bin" folder
-lib, *.dylib* -> ./bin # Copies all dylib files from packages lib folder to my local "bin" folder
\ No newline at end of file
+lib, *.dylib* -> ./bin # Copies all dylib files from packages lib folder to my local "bin" folder
```

Comment:
same

Reply:
Sorry. This was the code formatter. Shall I revert this?